libevm/contracts/ForgerStakes.sol

```diff
@@ -1,4 +1,4 @@
-// SPDX-License-Identifier: GPL-3.0
+// SPDX-License-Identifier: MIT
```

Comment:
a comment for the `struct Stake` below.
It is a bit inconsistent with the actual one: vrfPublicKey is 33 bytes

Reply:
True, but unfortunately there is no type `bytes33` in Solidity. The ABI serializer on our end also handles it like that, see [com.horizen.account.state.ForgerPublicKeys#vrfPublicKeyToAbi](https://github.com/HorizenOfficial/Sidechains-SDK/blob/d1180fd0160dd4adbd65de570fb2a3e293d8b2ea/sdk/src/main/scala/com/horizen/account/state/ForgerStakeMsgProcessor.scala#L460).

Reply:
Oops, I mixed up the files. This is just my draft for the Forger Stakes contract before we decided to implement them in Java. This is not used anywhere and could be removed.

Reply:
I was talking about the Forger Stakes Contract interface in Solidity where the vrf public key is separated to a `bytes32` property and then one more `bytes1`.

Reply:
done